scripts/setup.sh

```diff
@@ -33,7 +33,7 @@ elif [ -x "$(command -v dnf)" ];   then
 fi
 
 # C++ SDK requires `apache-arrow` and `cmake`
-packagesNeeded='apache-arrow cmake flatbuffers'
+packagesNeeded='apache-arrow clang-format cmake flatbuffers'
```

Comment:
technically wrong since for _building_ you shouldn't need clang-format as it won't be invoked until you make changes to the code!

Reply:
wondering if the same is true for flatbuffers?
Maybe safer to have clang-format in here after all. What do you think @teh-cmc ?

Reply:
I agree, those shouldn't be needed unless you edit the flatbuffers definitions.